nltk/twitter/twitter_demo.py

```diff
@@ -0,0 +1,231 @@
+# -*- coding: utf-8 -*-
+# Natural Language Toolkit: Twitter client
+#
+# Copyright (C) 2001-2015 NLTK Project
+# Author: Ewan Klein <ewan@inf.ed.ac.uk>
+# URL: <http://nltk.org/>
+# For license information, see LICENSE.TXT
+
+"""
+Examples to demo the :py:mod:`twitterclient` code.
```

Comment:
I suggest we add a line to the makefile which would convert the .ipynb file to .html and move it to the `howto` directory. But we should also think about how to publish iPython notebooks as HOWTOs.